src/checkers/inference/model/SourceVariableSlot.java

```diff
@@ -56,4 +56,7 @@ public boolean isInsertable() {
         return insertable;
     }
 
+    public void setInsertable(boolean insertable) {
```

Comment:
Can you add a comment that usage is discouraged and that we should remove the current uses of this method?